bindings/objective-c/librpc/librpc.m

```diff
@@ -544,7 +552,11 @@ - (nonnull RPCListenHandle *)observeProperty:(NSString *)name
         cb([[RPCObject alloc] initFromNativeObject:v]);
     });
 
-    NSAssert(cookie != NULL, @"rpc_connection_watch_property() failure");
+    if (!cookie) {
+        NSLog(@"rpc_connection_watch_property() unavailable");
```

Comment:
This should use `rpc_get_last_error()` to properly communicate error back to the user (either with `NSError *` output parameter or by raising exception)

Reply:
See below